clash-lib/prims/verilog/GHC_Prim.primitives.yaml

```diff
@@ -1395,7 +1395,7 @@
       genvar ~SYM[18];
       ~GENERATE
       for (~SYM[18]=0;~SYM[18]<32;~SYM[18]=~SYM[18]+1) begin : ~SYM[19]
-        assign ~SYM[1][~SYM[18]] = ~ARG[0][~SYM[18]];
+        assign ~SYM[1][~SYM[18]] = ~VAR[][0][~SYM[18]];
```

Comment:
Will `~VAR[]` just generate a random name?

Reply:
Yes, it will make up some unique name